### PR TITLE
fix: Reduce ephemeral upper bound from 365.25 days to 365 days

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/auth/AuthService.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ z.auth.AuthService = class AuthService {
   static get CONFIG() {
     return {
       POST_ACCESS_RETRY_LIMIT: 10,
-      POST_ACCESS_RETRY_TIMEOUT: 500,
+      POST_ACCESS_RETRY_TIMEOUT: z.util.TimeUtil.UNITS_IN_MILLIS.SECOND * 0.5,
```
Comment (ffflorian) [inline app/script/auth/AuthService.js:29]: I'd prefer
```js
POST_ACCESS_RETRY_TIMEOUT: z.util.TimeUtil.UNITS_IN_MILLIS.SECOND / 2,
```

File: app/script/auth/component/ClientList.jsx [hunk 2]
```diff
@@ -55,7 +55,8 @@ class ClientList extends React.Component {
 
   removeClient = (clientId, password) => {
     this.setState({showLoading: true});
-    this.loadingTimeout = window.setTimeout(this.resetLoadingSpinner.bind(this), 1000);
+    const timeout = z.util.TimeUtil.UNITS_IN_MILLIS.SECOND;
+    this.loadingTimeout = window.setTimeout(this.resetLoadingSpinner.bind(this), timeout);
```
Comment (ffflorian) [inline app/script/auth/component/ClientList.jsx:59]: Let's put it in one line

File: app/script/service/BackendClient.js [hunk 3]
```diff
@@ -27,10 +27,10 @@ z.service.BackendClient = class BackendClient {
     return {
       CONNECTIVITY_CHECK: {
         INITIAL_TIMEOUT: 0,
-        RECHECK_TIMEOUT: 2000,
-        REQUEST_TIMEOUT: 500,
+        RECHECK_TIMEOUT: z.util.TimeUtil.UNITS_IN_MILLIS.SECOND * 2,
+        REQUEST_TIMEOUT: z.util.TimeUtil.UNITS_IN_MILLIS.SECOND * 0.5,
```
Comment (ffflorian) [inline app/script/service/BackendClient.js:31]: Here I'd also prefer `z.util.TimeUtil.UNITS_IN_MILLIS.SECOND / 2`.

File: app/script/event/EventRepository.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ window.z.event = z.event || {};
 z.event.EventRepository = class EventRepository {
   static get CONFIG() {
     return {
-      E_CALL_EVENT_LIFETIME: 30 * 1000, // 30 seconds
+      E_CALL_EVENT_LIFETIME: 30 * z.util.TimeUtil.UNITS_IN_MILLIS.SECOND, // 30 seconds
```
Comment (bennycode) [inline app/script/event/EventRepository.js]: I think we can remove the comment now. :)

File: app/script/tracking/EventTrackingRepository.js [hunk 5]
```diff
@@ -34,7 +34,7 @@ z.tracking.EventTrackingRepository = class EventTrackingRepository {
     return {
       ERROR_REPORTING: {
         API_KEY: RAYGUN_API_KEY,
-        REPORTING_THRESHOLD: 60 * 1000, // milliseconds
+        REPORTING_THRESHOLD: z.util.TimeUtil.UNITS_IN_MILLIS.MINUTE, // milliseconds
```
Comment (bennycode) [inline app/script/tracking/EventTrackingRepository.js]: Comment can be removed

**Pull request closed without merging**